app/failed_logins/rest.py

```diff
@@ -0,0 +1,84 @@
+from flask import Blueprint, jsonify
+
+from app.dao.failed_logins_dao import (
+    dao_create_failed_login_for_ip,
+    dao_get_count_of_all_failed_logins_for_ip,
+    dao_get_failed_logins,
+    dao_get_latest_failed_login_by_ip,
+)
+from app.errors import InvalidRequest, register_errors
+from app.utils import (
+    calculate_delay_period,
+    check_request_within_throttle_period,
+    get_ip_address,
+    log_throttled_login,
+)
+
+failed_logins_blueprint = Blueprint(
+    "failed_logins",
+    __name__,
+    url_prefix="/failed-logins",
+)
+
+register_errors(failed_logins_blueprint)
+
+
+@failed_logins_blueprint.route("fetch-all")
+def get_all_failed_logins():
+    """
```

Comment:
This could end up returning hundreds of thousands of rows, is there any particular reason we need a get all failed logins functionality?

Reply:
I have removed this function as it was unused throughout.